ignite/templates/app/stargate/app/simulation_test.go.plush

```diff
@@ -5,16 +5,12 @@ import (
 	"testing"
 	"time"
 
-	"github.com/cosmos/cosmos-sdk/baseapp"
-	"github.com/cosmos/cosmos-sdk/codec"
+	"<%= ModulePath %>/app"
```

Comment:
import ordering

Reply:
It's already separated from the others, do you want it at the very bottom ?

Reply:
yes

Reply:
fixed e87ca3dd04998b35d67c9cfb3fde832bf35fb38b